Makefile

```diff
@@ -356,8 +356,11 @@ remotesystem:
 
 .PHONY: localapiv2
 localapiv2:
+	# TODO Remove once VM's with dependency
+	./hack/install_dependencies.sh python3-docker
```

Comment:
I'm not in favor of platform-specific calls like this in the general-use `Makefile`.  Since the problem is specific to CI, and short-term, would you mind doing something like this instead:

```diff
diff --git a/contrib/cirrus/runner.sh b/contrib/cirrus/runner.sh
index 084b196a9..b7e7ab852 100755
--- a/contrib/cirrus/runner.sh
+++ b/contrib/cirrus/runner.sh
@@ -63,6 +63,12 @@ function _run_unit() {
 }

 function _run_apiv2() {
+    # TODO Remove once VM's with dependency
+    if [[ "$OS_RELEASE_ID" == "fedora" ]]; then
+        dnf install -y python3-docker
+    else
+        apt-get -qq -y install python3-docker
+    fi
     make localapiv2 |& logformatter
 }

```

Reply:
@cevich Happy to make the changes. Do rhel and centos use "$OS_RELEASE_ID" == "fedora" ?

Reply:
They don't but...we don't run RHEL or CentOS in Cirrus-CI, only Ubuntu and Fedora :grinning: 

In fact (IIRC) this particular job only ever runs on Fedora.